src/main/java/org/jitsi/videobridge/transform/RtxTransformer.java

```diff
@@ -325,7 +396,7 @@ private boolean encapsulateInRtxAndTransmit(
         if (mediaStream != null)
         {
             rtxPkt.setSSRC((int) rtxSsrc);
-            rtxPkt.setPayloadType(channel.getRtxPayloadType());
+            rtxPkt.setPayloadType(rtxPayloadType);
```

Comment:
Unrelated to this PR (and I did take a personal note as well): we need to notify the cache when we retransmit a packet in RTX.